`Release,sole` failed, but with a different error:

```
ImportError: Failed to import test module: test_ometiffreader
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/opt/hudson/workspace/OME-FILES-CPP-DEV-merge-superbuild/BUILD_TYPE/Release/node/sole/ome-files-py/test/test_ometiffreader.py", line 32, in <module>
    import ome_files
ImportError: dlopen(/opt/hudson/workspace/OME-FILES-CPP-DEV-merge-superbuild/BUILD_TYPE/Release/node/sole/build/stage/lib/python2.7/site-packages/ome_files.so, 2): Library not loaded: libxalan-c.dylib
  Referenced from: /opt/hudson/workspace/OME-FILES-CPP-DEV-merge-superbuild/BUILD_TYPE/Release/node/sole/build/stage/lib/libome-common.5.3.2.dylib
  Reason: unsafe use of relative rpath libxalan-c.dylib in /opt/hudson/workspace/OME-FILES-CPP-DEV-merge-superbuild/BUILD_TYPE/Release/node/sole/build/stage/lib/libome-common.5.3.2.dylib with restricted binary
```

I.e., `ome_files.so` is found, but it cannot be imported due to the above xalan error in `libome-common.5.3.2.dylib`. /CC @rleigh-codelibre

--- a/helpers/cmake_environment.cmake
+++ b/helpers/cmake_environment.cmake
@@ -18,8 +18,7 @@ if(WIN32)
        "${OME_EP_TOOL_CACHE}/*/*/site-packages"
        "${OME_EP_TOOL_DIR}/*/site-packages"
        "${OME_EP_TOOL_DIR}/*/*/site-packages"
-       "${OME_EP_INSTALL_DIR}/lib/*/site-packages"
-       "${OME_EP_INSTALL_DIR}/lib64/*/site-packages")
+       "${OME_EP_INSTALL_DIR}/*/*/site-packages")
   foreach(dir ${python_dirs})
     file(TO_NATIVE_PATH "${dir}" dir)
     if(PYTHONPATH)
@@ -54,8 +53,7 @@ else()
        "${OME_EP_TOOL_CACHE}/*/*/site-packages"
        "${OME_EP_TOOL_DIR}/*/site-packages"
        "${OME_EP_TOOL_DIR}/*/*/site-packages"
-       "${OME_EP_INSTALL_DIR}/lib/*/site-packages"
-       "${OME_EP_INSTALL_DIR}/lib64/*/site-packages")
+       "${OME_EP_INSTALL_DIR}/*/*/site-packages")
   foreach(dir ${python_dirs})
     if(PYTHONPATH)
       set(PYTHONPATH "${dir}:${PYTHONPATH}")

--- a/packages/ome-files-py/build.cmake
+++ b/packages/ome-files-py/build.cmake
@@ -18,7 +18,7 @@ string(REPLACE ";" ":" INCLUDE_ARG "${INCLUDE_DIRS}")
 string(REPLACE ";" ":" LIB_ARG "${LIB_DIRS}")
 
 execute_process(COMMAND python setup.py build_ext
-  "-I${INCLUDE_ARG}" "-L${LIB_ARG}" "-R${LIB_ARG}"
+  "-I${INCLUDE_ARG}" "-L${LIB_ARG}"
   WORKING_DIRECTORY "${SOURCE_DIR}"
   RESULT_VARIABLE build_ext_result)
 if(build_ext_result)

